integration/spark-common/src/main/scala/org/apache/spark/sql/optimizer/CarbonDecoderOptimizerHelper.scala

```diff
@@ -84,7 +87,16 @@ class CarbonDecoderProcessor {
         }
         nodeList.add(ArrayCarbonNode(nodeListSeq))
       case e: UnaryNode => process(e.child, nodeList)
-      case i: InsertIntoTable => process(i.child, nodeList)
+      case i: InsertIntoTable =>
+        var sparkVersion21: Boolean = false
```

Comment:
here, please give one comment to explain : sparkVersion21 is for spark 
2.1.0.  because in the future there are may many spark versions.

Reply:
Done